lbrynet/core/server/DHTHashAnnouncer.py

```diff
@@ -9,14 +9,37 @@
 log = logging.getLogger(__name__)
 
 
+class DequeSet(object):
+    def __init__(self):
+        self._queue = collections.deque()
+        self._items = set()
+
+    def append(self, item):
```

Comment:
because the items being added are (h, announce_deferred), they aren't actually unique.  Should turn this into a special purposed data structure that is called like: `def append(self, blob_hash, announce_deferred)`
and have `_items` be a set of blobs, and `_queue` be a queue of (h, deferred) tuples.